src/main/java/gregtechfoodoption/GTFOMaterialHandler.java

```diff
@@ -510,6 +512,126 @@ public class GTFOMaterialHandler {
     public static final Material EnergizedCoffee = fluidBuilderFood(21991, "energized_coffee", 368)
             .color(0x695934)
             .build();
+    public static final Material DiethylEther = fluidBuilder(21993, "Diethyl_Ether")
+            .color(0xFFFFFF)
+            .build();
+    public static final Material BromoBenzene = fluidBuilder(21994, "Bromo_Benzene")
+            .color(0xFFFFE4)
+            .build();
+    public static final Material Acetylene = fluidBuilder(21995, "Acetylene")
+            .color(0xFFFFFF)
+            .build();
+    public static final Material Formaldehyde = fluidBuilder(21996, "Formaldehyde")
+            .color(0x00B140)
+            .build();
+    public static final Material onefourButynediol = fluidBuilder(21997, "onefourButynediol")
+            .color(0xFFFFFF)
+            .build();
+    public static final Material onefourButanediol = fluidBuilder(21998, "onefourButanediol")
+            .color(0xFFFFFF)
+            .build();
+    public static final Material Methylamine = fluidBuilder(22000, "Methylamine")
+            .color(0xFFFFFF)
+            .build();
+    public static final Material yButyrolactone = fluidBuilder(22002, "γButyrolactone")
+            .color(0xFFFFFF)
+            .build();
+    public static final Material yButyrolactoneammonia = fluidBuilder(22003, "γButyrolactoneammonia")
+            .color(0xFFFFFF)
+            .build();
+    public static final Material cooledyButyrolactoneammonia = fluidBuilder(22004, "cooledγButyrolactoneammonia")
+            .color(0xFFFFFF)
+            .build();
+    public static final Material twoPyrrolidone = fluidBuilder(22005, "twoPyrrolidone")
+            .color(0xFFFFFF)
+            .build();
+    public static final Material aqueoustwoPyrrolidone = fluidBuilder(22006, "aqueoustwoPyrrolidone")
```

Comment:
Could you please make sure to capitalize these materials here?

Reply:
bro what why those specific ones

Reply:
OK actually I should probably just format it myself rather than try to give a lecture on how I prefer the formatting

Reply:
this goes crazy